### ISSUE TITLE
No scroll in case of user scroll

### ISSUE THREAD

**Issue by simzer** [vizzuhq/ipyvizzu#70]
If the notebook is running and the user scrolls, then auto scrolling should switch off for that run.
It is inconvenient for the user that they couldn't scroll during running.

Something like this would be the solution:

global scope:
```javascript
let inhibitScroll = false;
document.addEventListener('scroll', function(e) {
  inhibitScroll = true;
});
```

cell:
```javascript
if (!inhibitScroll) myVizzu.scrollIntoView(...);
```

I don't how to set `inhibitScroll` back to `false` after running, but next run should do autoscrolling again.

**Comment by simzer** [comment]
@veghdev Since you pointed out that scrollIntoView() triggers scroll event, the improved solution would be st like this:
```javascript
let inhibitScroll = false;
let ipyvizzuScrolles = false;

document.addEventListener('scroll', function(e) {
  if (ipyvizzuScrolles) {
    ipyvizzuScrolles = false;
  }
  else
  {
    inhibitScroll = true;
  }
});

// cell
if (!inhibitScroll) {
  ipyvizzuScrolles = true;
  myVizzu.scrollIntoView(...);
}
```

**Comment by simzer** [comment]
@veghdev Maybe an even better solution would be (since the browser can also trigger scroll event on various occasions) to explicitly detect user scrolling via the event type:

```javascript
let inhibitScroll = false;
document.addEventListener('scroll', function(evt) {
    if (evt.type === 'keyup' || evt.type === 'mousewheel')
      inhibitScroll = true;
});

if (!inhibitScroll) myVizzu.scrollIntoView(...);
```

**Comment by veghdev** [comment]
fixed in #82
